website/server/models/group.js

```diff
@@ -85,6 +85,7 @@ export const schema = new Schema({
     $type: String, enum: ['private', 'public'], default: 'private', required: true,
   },
   chat: Array, // Used for backward compatibility, but messages aren't stored here
+  bannedWordsAllowed: { $type: Boolean, required: false },
```

Comment:
could you add this field to the list of un-settable fields here https://github.com/HabitRPG/habitica/pull/12440/files#diff-b6f239870392ed0ec2a2b73cbebaa699L153 (same file line 153) to make sure it cannot be set when the group is created or updated

Reply:
Sure thing, I can get this change done later today :)